sites/main-site/public/_redirects

```diff
@@ -40,7 +40,7 @@
 /docs/usage/offline                                          https://nf-core-docs.netlify.app/docs/usage/getting_started/offline 200
 /docs/usage/troubleshooting                                  https://nf-core-docs.netlify.app/docs/usage/troubleshooting/overview 200
 /docs/usage/tutorials                                        https://nf-core-docs.netlify.app/docs/tutorials 200
-/docs/usage/tutorials/step_by_step_institutional_profile     https://nf-core-docs.netlify.app/docs/tutorials/use_nf-core_pipelines/config_institutional_profile 200
+/docs/usage/tutorials/step_by_step_institutional_profile     https://nf-core-docs.netlify.app/docs/tutorials/use_nf-core_pipelines/writing_institutional_profile 200
```

Comment:
Should also add a new redirection rule as well if we are removing `/config_institutional_profile`

Reply:
Like so @ewels ? (see below)